webui/scheduled.ego

```diff
@@ -40,6 +40,7 @@ func ego_listScheduled(w io.Writer, req *http.Request, set storage.SortedSet, co
             <th><%= t(req, "When") %></th>
             <th><%= t(req, "Queue") %></th>
             <th><%= t(req, "Job") %></th>
+            <th><%= t(req, "Priority") %></th>
```

Comment:
Please add an entry at the bottom of en.yml for this new UI string, so other languages will know to add it.